mlir/include/mlir/Dialect/Tosa/Transforms/Passes.h

```diff
@@ -63,7 +63,7 @@ struct ValidationOptions {
   }
 };
 
-std::unique_ptr<Pass> createTosaValidationPass(
+std::unique_ptr<OperationPass<ModuleOp>> createTosaValidationPass(
     ValidationOptions const &options = ValidationOptions());
```

Comment:
Nit: this should be auto-generated: just remove the `let constructor =` from the .td file

Reply:
done